fusor-ember-cli/app/models/openstack-deployment.js

```diff
@@ -39,6 +39,8 @@ export default DS.Model.extend(ValidatedModel, {
   undercloud_ssh_username: DS.attr('string'),
   undercloud_ssh_password: DS.attr('string'),
 
+  overcloud_deployed: DS.attr('boolean'),
```

Comment:
@cfchase, for booleans to I tend to prefix with `is_` so it would be `is_overcloud_deployed`. We haven't followed this except for `is_disconnected`, but I think it's a good practice to follow.